docker/rust_dockerfile

```diff
@@ -8,11 +8,13 @@ RUN yum install -y epel-release
 RUN yum makecache \
 	&& yum update -y \
 	&& yum install -y tar wget git which file unzip python-pip \
-	&& yum install -y make cmake gcc gcc-c++ libstdc++-static pkg-config psmisc \
+	&& yum install -y make cmake3 gcc gcc-c++ libstdc++-static pkg-config psmisc \
 	&& yum install -y libzstd-devel lz4-devel zlib-devel bzip2-devel snappy-devel libdwarf-devel elfutils-libelf-devel elfutils-devel binutils-devel \
```

Comment:
seem we can remove libzstd-devel lz4-devel zlib-devel bzip2-devel snappy-devel now